website_sale_product_trial/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Form to order subscription product
```

Comment:
this doesn’t match the module’s summary.

Reply:
changed